r/R/dplyr-functions.R

```diff
@@ -848,6 +844,21 @@ nse_funcs$wday <- function(x,
   Expression$create("day_of_week", x, options = list(count_from_zero = FALSE, week_start = week_start))
 }
 
+nse_funcs$is.Date <- function(x) {
+  inherits(x, "Date") ||
+    (inherits(x, "Expression") && x$type_id() %in% Type[c("DATE32")])
+}
+
+nse_funcs$is.instant <- nse_funcs$is.timepoint <- function(x) {
+  inherits(x, c("POSIXt", "POSIXct", "POSIXlt", "Date")) ||
+    (inherits(x, "Expression") && x$type_id() %in% Type[c("TIMESTAMP", "DATE32")])
+}
+
+nse_funcs$is.POSIXct <- function(x) {
+  inherits(x, "POSIXct") ||
+    (inherits(x, "Expression") && x$type_id() %in% Type[c("TIMESTAMP")])
+}
+
```

Comment:
Should we add a binding for `is.POSIXt()`? It's defined in lubridate as `inherits(x, "POSIXt")`

Reply:
Likewise with `is.POSIXlt()` which is defined in lubridate as `inherits(x, "POSIXlt")`

Reply:
Oh, I didn't see your other comments about this. Disregard this.